tests/functional.rs

```diff
@@ -93,9 +73,8 @@ async fn set_up_ca_with_repo(ca: &Handle) {
     let response = embedded_repository_response(ca).await;
 
     // Update the repo for the child
-    let update = RepositoryUpdate::Rfc8181(response);
-    repo_update(ca, update).await;
-    assert!(repo_ready(ca).await);
```

Comment:
WHy is this check no longer needed?

Reply:
We don't need to wait for it. It is implicitly tested when we wait for things to be published.